pvlib/iam.py

```diff
@@ -121,6 +122,10 @@ def physical(aoi, n=1.526, K=4., L=0.002):
         indicates that 0.002 meters (2 mm) is reasonable for most
         glass-covered PV panels.
 
+    n_ar : numeric, default None
```

Comment:
```suggestion
    n_ar : numeric, optional
```
See https://github.com/pvlib/pvlib-python/issues/1574

Reply:
I updated the docstring.